ajax/libs/onsen/package.json

```diff
@@ -23,17 +23,14 @@
     "type": "git",
     "url": "https://github.com/OnsenUI/OnsenUI.git"
   },
-  "autoupdate": {
-    "source": "git",
-    "target": "git://github.com/OnsenUI/OnsenUI.git",
-    "fileMap": [
-      {
-        "basePath": "build",
-        "files": [
-          "js/*",
-          "css/*"
-        ]
-      }
-    ]
-  }
+  "npmName": "onsenui",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "js/*",
+        "css/*"
```

Comment:
Thank you, but `onsenui.css` has `@import` such as `@import url("font_awesome/css/font-awesome.min.css");`, so I think `"+(js|css)/**/*` is the correct value in this case.